src/frontend/screens/Settings/ProjectSettings/YourTeam/SelectDevice.tsx

```diff
@@ -91,4 +111,45 @@ const styles = StyleSheet.create({
     padding: 20,
     paddingTop: 40,
   },
+  deviceListContainer: {
+    gap: 10,
+  },
 });
+
+/**
+ * Applies specialized, context-specific behavior on top of `useLocalPeers()` in the following ways:
+ *
+ * - State is bound to the lifetime of the consuming component i.e. gets re-initialized during remounts
+ * - Only connected peers are ever *added* to the state. However, subsequent updates to those peers (e.g. disconnecting) are still reflected in the state.
+ */
+function useInitiallyConnectedPeers() {
```

Comment:
naming is a little confusing but hopefully these comments help clarify what this hook is doing. Basically it's keeping its own cache/state that's tied to the consuming components lifecycle, as opposed to the global state (which `useLocalPeers()` is used for)